packages/backend/src/database/migrations/20240220102923_add-weight-to-spaces-vector.ts

```diff
@@ -0,0 +1,62 @@
+import { Knex } from 'knex';
+import { SpaceTableName } from '../entities/spaces';
+
+const customSearchConfigName = `lightdash_english_config`;
+
+export async function up(knex: Knex): Promise<void> {
+    // drop search_vector column if it exists - don't think I can alter it since it is generated
+    if (await knex.schema.hasColumn(SpaceTableName, 'search_vector')) {
+        await knex.schema.alterTable(SpaceTableName, (table) => {
+            table.dropColumn('search_vector');
+        });
+    }
+
+    // add search_vector column weighted by name
+    await knex.raw(`
+      ALTER TABLE ${SpaceTableName} ADD COLUMN search_vector tsvector
+      GENERATED ALWAYS AS (
+          setweight(to_tsvector('${customSearchConfigName}', coalesce(name, '')), 'A')
```

Comment:
we can skip `coalesce(name, '')` as it does not do anything. column constraint is set to not null